### PR TITLE
Update java.md

### PR DESCRIPTION
**Pull request by mssindhurid**
Azure Attestation Data-plane SDKs are being released for dotnet, python and JS. As recommended by Data-plane SDK team, adding comment to notify customers that Java SDK will be coming soon

### CODE REVIEW

**Comment by mssindhurid** [comment]
#sign-off

**Review by LarryOsterman — APPROVED** [review]

File: releases/2021-01/java.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The Azure SDK team is pleased to announce our January 2021 client library releas
 
 #### Beta
 
-- _Add packages_
+- Azure.Security.Attestation (coming soon)
```
Comment (LarryOsterman) [inline releases/2021-01/java.md:21]: Should probably either be Microsoft Azure Attestation(coming soon) or `azure.security.attestation` (the name of the new types).

Comment (alzimmermsft) [reply releases/2021-01/java.md:21]: I would recommend the first option as we generally use the name of the library here, such as `Azure Core`

**Comment by mssindhurid** [comment]
@czubair  - Can you please review the changes to notify customers on the plan for releasing Java SDK for Azure Attestation

File: releases/2021-01/java.md (hunk 1 above)
Comment (mssindhurid) [reply releases/2021-01/java.md]: @alzimmermsft  - Updated to Microsoft Azure Attestation(coming soon)

**Review by alzimmermsft — APPROVED** [review]

**Pull request merged**